### PR TITLE
Initial experimental version

### PR DESCRIPTION
**Pull request by onishiweb**

### CODE REVIEW

File: main.scss [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+	@include oNormaliseHTML;
+
+	.o-normalise-clearfix {
+		@include oNormaliseClearfix;
+	}
+
+	.o-normalise-visually-hidden {
+		@include oNormaliseVisuallyHidden;
+	}
+
```
Comment (rowanmanning) [inline main.scss:11]: no `.o-normalise-box-sizing`? Sure there's a reason, but just curious

**Review by rowanmanning — APPROVED** [review]
Looks good to me. One comment

File: main.scss (hunk 1 above)
Comment (onishiweb) [reply main.scss:11]: Because it's being included in the `oNormaliseHTML` mixin by default. Also this is just for the `non-silent` mode as well

**Comment by onishiweb** [comment]
Moved this to #2

**Pull request closed without merging**